src/main/reactjs/src/tests/cypress/integration/public_translator.spec.ts

```diff
@@ -64,11 +64,19 @@ describe('PublicTranslatorFilters', () => {
     onToast.expectText('Voit valita vain yhden kieliparin yhteydenottoon');
   });
 
-  it('it should show a toast notification when language pair is defined, a translator is selected, and user tries to change to lang', () => {
-    onPublicTranslatorFilters.filterByLanguagePair('suomi', 'ruotsi');
-    onPublicTranslatorsListing.clickTranslatorRow('1940');
-    onPublicTranslatorFilters.clickToLang();
```

Comment:
Lähtikö tämä tarkoituksella pois? Kommentoin siis aiemmin siihen liittyen että tämä kyseinen blokki oli tuplana, mutta nyt lähti myös tämä jo olemassa ollut blokki pois. Ei sillä ehkäpä niin väliä, mutta ajattelinpa vielä varmistaa.

Reply:
joo, katsoin, että sama mutta tämähän oli to lang, ei from lang, palautetaan ->